_includes/sections/parallax.html

```diff
@@ -0,0 +1,50 @@
+{% comment %} 
+<!--
+  Parallax Section
+
+  This partial is used for parallax sections.  Currently used on the Why We
+  Care page.
+  
+--> 
+{% endcomment %}
+
+{% assign laxes = site.sections |where:"type","parallax" |where:"title",include.title %}
+
+{% for lax in laxes %}
+
+
+<div id="weCareBecause" class="section parallax nobottommargin" data-stellar-background-ratio="0.4" style="background-image: url({{ lax.bg-img }});">
```

Comment:
Can't have an `id` in a for loop.

Reply:
Good catch!  This is fixed in bf288da1453f